### DISCUSSION TITLE
[Feature] Auto fill in arrival date

### DISCUSSION THREAD

**Discussion by BrainPonders** [johanohly/AirTrail#33] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The majority of the flights in Europe are all within the same day. Having the arrival date auto fill in by copying of the departure date would safe a step for most flights. In other situations I can just correct the date.

**Comment by johanohly** [comment]
In the next release, when no arrival date is set, but an arrival time is set, the same date will be used. Not sure why I didn't do this to begin with.

**Reply by BrainPonders** [reply]
Great! I will wait for it before I start to insert all previous flights from the last 24 months manually. Btw, perhaps you would also consider to drop the : between de hours in minutes and allow me to enter 2245 or 0615 instead of 22:45 and 06:45.

**Reply by johanohly** [reply]
I guess I can do that and while I'm at it allow `.` instead of `:` :+1:

**Reply by BrainPonders** [reply]
Nice :-)

**Comment by johanohly** [comment]
Added in v0.1.2
